src/software/simulated_tests/ball_is_off_ground.py

```diff
@@ -0,0 +1,55 @@
+import software.python_bindings as tbots_cpp
+from proto.import_all_protos import *
+
+from software.simulated_tests.validation import (
+    Validation,
+    create_validation_geometry,
+    create_validation_types,
+)
+
+
+class BallIsOffGround(Validation):
+
+    """Checks if a ball is of ground (i.e. it has air time)"""
+
+    def __init__(self, threshold=0.01):
+        self.threshold = threshold
+
+    def get_validation_status(self, world) -> ValidationStatus:
+        """Checks if the ball has is threshold meters off the ground
+
+        :param world: The world msg to validate
+        :returns: FAILING until a ball has a positive threshold distance off the ground
+                  PASSING when the ball has a positive threshold distance off the ground
+        """
+        if world.ball.current_state.distance_from_ground > self.threshold:
+            return ValidationStatus.PASSING
+        else:
+            return ValidationStatus.FAILING
+
+    def get_validation_geometry(self, world) -> ValidationGeometry:
+        """Returns the underlying geometry this validation is checking
+
+        :param world: The world msg to create validation geometry from
+        :returns: ValidationGeometry containing geometry to visualize
+
+        """
+        # TODO: Make this a nicer visualization
```

Comment:
nit: remaining TODO

Reply:
I left this as a note in the PR. It isn't an urgent TODO since it is purely an aesthetic update. The purpose is to have some way to visualize the vertical nature of this test as I don't think our current shapes protos are equipped with rendering things in the z dimension. What I have in place is just a circle around the ball. I will create an issue for this and I think it could be an interesting/creative mid-difficulty ticket for upcoming members.

Reply:
To avoid having TODOs forgotten in our code, we have a requirement that also TODOs must be attributed with a specific github issue (general notation in code is `# TODO (#ticket_number): high level description`). Or we just end up removing the TODO if it's not significant.